jobs/JGLOBAL_WAVE_POST_BNDPNT

```diff
@@ -9,9 +9,14 @@ export errchk=${errchk:-err_chk}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_STATION
-
-if [[ ! -d ${COM_WAVE_STATION} ]]; then mkdir -p "${COM_WAVE_STATION}"; fi
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+   COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+   COMIN_WAVE_HISTORY:COM_WAVE_HISTORY_TMPL \
+   COMOUT_WAVE_STATION:COM_WAVE_STATION_TMPL
+
+for out_dir in "${COMOUT_WAVE_HISTORY}" "${COMOUT_WAVE_STATION}"; do
```

Comment:
`COMOUT_WAVE_HISTORY` is undefined and should be a `COMOUT` for this job.

Reply:
That directory is only read by this job.

Reply:
So, it should be a `COMIN` as seen on line 14 and should be removed from this line.

Reply:
@HenryWinterbottom-NOAA 
This still needs to be resolved.

Reply:
https://github.com/HenryWinterbottom-NOAA/global-workflow/blob/ccc071820383a9dcf73fc9927877a34e0db459bc/jobs/JGLOBAL_WAVE_POST_BNDPNT#L17

I believe that it is.